src/components/RenderTime.tsx

```diff
@@ -31,76 +24,36 @@ function getCountdownText({
   switch (countdownSettings.source) {
     case ClockSource.input:
       if (!countdownSettings.endCountdownTime) {
-        return countdownSettings.endText;
+        return countdownSettings.noValueText;
```

Comment:
even though this makes sense it is best to not change this existing behaviour or people will get unexpected behaviours when they update their plugin.